### PR TITLE
Add customer testing kb

### PR DESCRIPTION
**Pull request by ChristineSala**

### CODE REVIEW

File: Security/customer-testing-clc.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+|**Penetration Testing**   |Med                   |May run into automated throttling and possibly be shut down due to abuse monitoring if performed outside customer’s own VLAN.|
+|**Vulnerability Testing** |Low                   |No impact to other customers.                                                                                                |
+
+## Detailed Information
+
+### Load Testing
+
+Any test intended to find either limits or performance degradation points in systems.  
+
+**Required Notification:** CenturyLink does not allow customers to perform load testing except in very special, highly controlled circumstances that must be closely coordinated ahead of time.  Advance notice to the CLC help desk (in all cases) by opening a ticket at help@ctl.io and coordinating with a customer service associate (CSA) during the testing.
```
Comment (chrislittle) [inline Security/customer-testing-clc.md]: CSA is a completely different team (Cloud Solutions Architects).  may want to reword this to the NOC titles.  I believe they are Customer Care Engineer for cloud.  We don't want people confusing our pre-sales architect teams (the team i'm on) with support.

File: Security/customer-testing-clc.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{{{ "title": "Customer Testing on CLC",
+        "date": "05-23-2016",
+        "author": "Platform Security",
+        "attachments": [],
+        "contentIsHTML": false,
+        "sticky": false }}}
+
+## Description
```
Comment (srveit) [inline Security/customer-testing-clc.md]: Change to `###`

File: Security/customer-testing-clc.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+There are three types of testing that we see our customers wanting or needing to perform: load testing, penetration testing, and vulnerability scanning. Each type has its own potential impact to the platform, which is why we have requirements for notifying us ahead of time, during the testing process and afterward.
+
+## Audience
```
Comment (srveit) [inline Security/customer-testing-clc.md]: Change to `###`

File: Security/customer-testing-clc.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+CenturyLink Cloud Users
+
+## Impact
```
Comment (srveit) [inline Security/customer-testing-clc.md]: Change to `###`

File: Security/customer-testing-clc.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+|                          | **Potential Impact** | **Comments**                                                                                                                |
+|--------------------------|----------------------|-----------------------------------------------------------------------------------------------------------------------------|
+|**Load Testing**          |High                  |May run into automated throttling and may be shut down due to abuse monitoring on network bandwidth, CPU, or disk usage.     |
+|**Penetration Testing**   |Med                   |May run into automated throttling and possibly be shut down due to abuse monitoring if performed outside customer’s own VLAN.|
+|**Vulnerability Testing** |Low                   |No impact to other customers.                                                                                                |
+
+## Detailed Information
```
Comment (srveit) [inline Security/customer-testing-clc.md]: Change to `###`

**Comment by jwebs1** [comment]
@srveit I am helping Christine with this. She mentioned something about you not seeing the changes she made based on your comments, but I can see them. Is github having problems with versioning?

**Pull request merged**